AmazonAdvertisingApi/RequestPrefixes.php

```diff
@@ -0,0 +1,24 @@
+<?php
+namespace AmazonAdvertisingApi;
+
+require_once "Versions.php";
+
+
+class RequestPrefixes
+{
+	private $apiVersion = null;
+
+	public function __construct() 
+	{
+		$versions = new Versions();
+		$this->apiVersion = $versions->versionStrings["apiVersion"];
+	}
+
+    public $requestPrefixStrings = array(
+		"display" => "sd/",
+		"products" => "{$this->apiVersion}/sp/",
+		"brands"   => "sb/",
+		"profile" => "{$this->apiVersion}/",
+		"report" => "{$this->apiVersion}/"
+    );
+}
```

Comment:
You commit name indicated you don't like this?
Is this any better?
`public static function getPrefix($requestType)
{
switch ($requestType)
{
case 'profile':
case 'report':
return "{$this->apiVersion}/";
case 'products':
return "{$this->apiVersion}/sp/"}
}
}`

and so on. As below may be nicer if api version is just a constant.